### PR TITLE
Add testing and linting

### PR DESCRIPTION
**Pull request by m3nu**
- Run built-in Homebrew tests
- Lint formula file to pass `brew audit`
- Add some more tests for `borg mount`

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    name: Test Homebrew install
+    runs-on: macos-latest
+    timeout-minutes: 60
```
Comment (ThomasWaldmann) [inline .github/workflows/main.yml:7]: ah, nice, need that elsewhere also. :-)

File: Formula/borgbackup-fuse.rb [hunk 2]
```diff
@@ -55,4 +54,18 @@ class BorgbackupFuse < Formula
...
+    cp test_fixtures("test.pdf"), testpath
+    Dir.chdir(testpath) do
+      system "#{bin}/borg", "init", "-e", "none", "test-repo"
+      system "#{bin}/borg", "create", "--compression", "zstd", "test-repo::test-archive", "test.pdf"
+    end
+    mkdir testpath/"restore" do
+      system "#{bin}/borg", "extract", testpath/"test-repo::test-archive"
+    end
+    assert_predicate testpath/"restore/test.pdf", :exist?
+    assert_equal File.size(testpath/"restore/test.pdf"), File.size(testpath/"test.pdf")
```
Comment (ThomasWaldmann) [inline Formula/borgbackup-fuse.rb:69]: hmm, maybe one could also use `diff` or `cmp` or so to actually test if the contents are the same.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+          # brew audit --strict borgbackup-fuse.rb
+          brew install borgbackup-fuse.rb
+          brew test borgbackup-fuse.rb
+
+      - name: Test mount command
+        run: |
+          borg init -e none test-repo
+          borg create test-repo::1 Formula
+          mkdir mnt
+          borg mount test-repo::1 mnt
```
Comment (ThomasWaldmann) [inline .github/workflows/main.yml:32]: nitpick: maybe use `::test-archive` instead of `::1`?

**Comment by m3nu** [comment]
Thanks for reviewing. Addressed both issues.

File: .github/workflows/main.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Install macFUSE
+        run: brew install --cask osxfuse
+
+      # - name: Debug
+      #   uses: mxschmitt/action-tmate@v3
```
Comment (ThomasWaldmann) [inline .github/workflows/main.yml:17]: are you still working on that or is it a remainder that should be removed for now?

Comment (m3nu) [reply .github/workflows/main.yml:17]: I think it should stay there if someone wants to debug the workflow in the future.

Comment (ThomasWaldmann) [reply .github/workflows/main.yml:17]: ok

**Review by ThomasWaldmann — APPROVED** [review]
lgtm

**Comment by m3nu** [comment]
Thanks for reviewing! Merged.

**Pull request merged**